lib/types/multipart.js

```diff
@@ -29,6 +29,7 @@ function Multipart (boy, cfg) {
   const self = this
   let boundary
   const limits = cfg.limits
+  const isPartAFile = cfg.isPartAFile || ((fieldName, contentType, fileName) => (contentType === 'application/octet-stream' || fileName !== undefined))
```

Comment:
@cyantree We can't assume filename that is empty string to be a non-field by default?

Reply:
this is the old busboy behaviour. So we are non-breaking.

Reply:
fair.
we can delete `MIGRATING.md` file then, because there will be no migration involved :D

Reply:
Yep, that's good. :)